portfolio/src/main/webapp/script.js

```diff
@@ -62,3 +62,67 @@ function changeProfilePic() {
   document.getElementById('pfp-link').href = '/images/' + img;
   document.getElementById('pfp').src = '/images/' + img;
 }
+
+let xando = [
+  [0, 0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0, 0],
+];
+
+/**
+ * Process tic-tac-toe game.
+ */
+function ticTacToe(cell) {
+  const playerTurn = document.getElementById('player-turn').className;
```

Comment:
I think it would be cleaner to convert to a number here, so that you don't have to do it everywhere below.

Reply:
Done